BraintreeCore/src/main/java/com/braintreepayments/api/DeviceInspector.kt

```diff
@@ -107,12 +99,10 @@ class DeviceInspector @VisibleForTesting internal constructor(
             } catch (ignored: PackageManager.NameNotFoundException) { null }
         }
 
-    private fun getUserOrientation(context: Context?): String =
-        when (context?.resources?.configuration?.orientation ?: Configuration.ORIENTATION_UNDEFINED) {
-            Configuration.ORIENTATION_PORTRAIT -> "Portrait"
-            Configuration.ORIENTATION_LANDSCAPE -> "Landscape"
-            else -> "Unknown"
-        }
+    private fun getAPIVersion(): String {
+        val sdkInt = Build.VERSION.SDK_INT
+        return "Android API $sdkInt"
```

Comment:
Would it be easier on the reporting end if we didn't prepend `Android API`? Just sending the int value might make it easier to filter data for a given range of API versions.

Reply:
It would be easier, yeah, but on iOS, we send **_iOS X.X_** I'm not sure about the query stuff, though. We could check with the analytics team to see if it works better for them without the platform prefix. 
And make the change on both platforms in a separate PR.

Reply:
Yeah if we're sending platform (iOS/Android) in a separate param, I think it'd be cleaner to send the OS version as an int. Agreed in a separate PR.